### PR TITLE
update buildkit and spdx deps

### PR DESCRIPTION
**Pull request by deitch**
Two things:

1. Updates the buildkit and other dependencies for `dockerfile-add-scanner`
2. Fixes the Dockerfile so it calls make, thus ensuring it gets the correct build arguments

### CODE REVIEW

**Comment by deitch** [comment]
cc @mikem-zed

File: tools/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,21 @@
 # Use an official Golang runtime as a parent image
-FROM golang:1.20.1 as builder
+FROM golang:1.20.10-alpine3.18 as builder
+
+RUN apk add make
```
Comment (eriknordmark) [inline tools/Dockerfile:4]: Looks like yetus wants you to specify a version.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Comment by mikem-zed** [comment]
LGTM. All issues I've found seems to be addressed. I have not tested this pr myself yet. But I'm sure Avi did

**Comment by deitch** [comment]
> But I'm sure Avi did

Certainly did.

**Comment by deitch** [comment]
Addressed the yetus issues. Needs a new approval, I think.

**Comment by deitch** [comment]
yetus has no real errors on the code, just this one:

```
  ERROR: Failed to write github status. Token expired or missing repo:status write?
```

Look familiar @eriknordmark ?

**Comment by eriknordmark** [comment]
> yetus has no real errors on the code, just this one:
> 
> ```
>   ERROR: Failed to write github status. Token expired or missing repo:status write?
> ```
> 
> Look familiar @eriknordmark ?

We've seen those before but i think there was normally also some other error earlier in the yetus log. Anyhow let's merge this

**Pull request merged**